test/postgresql.test.js

```diff
@@ -206,6 +206,15 @@ describe('postgresql connector', function () {
           console.warn.restore();
         });
 
+        it('should work', function(done) {
+          Post.find({where: {content: {regexp: '^a/i'}}}, function(err, posts) {
```

Comment:
For regex strings it is. My toRegExp normalization will convert it to a RegExp object properly (which in turn converts '^a/i' to /a/i underneath the hood.